_overviews/scala3-book/introduction.md

```diff
@@ -12,8 +12,8 @@ The goal of this book is to provide an informal introduction to the Scala langua
 It touches on all Scala topics, in a relatively light manner.
 If at any time while you’re reading this book and want more information on a specific feature, you’ll find links to our [_Reference_ documentation][reference], which covers many new features of the Scala language in more detail.
 
-Over the course of this book, we hope to demonstrate that Scala is a beautiful, expressive programming language, with a clean, modern syntax, and supports functional programming (FP), object-oriented programming (OOP), and a fusion of FP and OOP in a typed setting.
-Scala’s syntax, grammar, and features have been re-thought, debated in an open process, and updated in 2020 to be more clear and easier to understand than ever before.
+Over the course of this book, we hope to demonstrate that Scala is a beautiful, expressive programming language, with a clean, modern syntax, which supports functional programming (FP) and object-oriented programming (OOP), and that provides a safe static type system.
```

Comment:
I thought that “fusion of FP and OOP in a typed setting” was a bit redundant with the previous mentions of FP and OOP, so I’ve changed the last point to highlight only the static type system.